res/css/views/rooms/_ReadReceiptGroup.scss

```diff
@@ -0,0 +1,132 @@
+.mx_ReadReceiptGroup {
+    position: relative;
+    display: inline-block;
+    // This aligns the avatar with the last line of the
+    // message. We want to move it one line up
+    // See .mx_GroupLayout .mx_EventTile .mx_EventTile_line in _GroupLayout.scss
+    top: calc(-$font-22px - 3px);
+    user-select: none;
+    z-index: 1;
+
+    .mx_ReadReceiptGroup_button {
+        display: inline-flex;
+        flex-direction: row;
+        height: 16px;
+        padding: 4px;
+        border-radius: 6px;
+
+        &.mx_AccessibleButton {
+            &:hover {
+                background: #F4F6FA;
+            }
+        }
+    }
+
+    .mx_ReadReceiptGroup_remainder {
+        color: #737D8C;
+        font-size: $font-11px;
+        line-height: $font-16px;
+        margin-right: 8px;
+    }
+
+    .mx_ReadReceiptGroup_container {
+        position: relative;
+        display: block;
+        height: 100%;
+        width: 36px;
+
+        .mx_BaseAvatar {
+            position: absolute;
+            display: inline-block;
+            height: 14px;
+            width: 14px;
+            border: 1px solid #fff;
```

Comment:
we should use a variable here, as this is probably not desirable for the dark theme